integrated_channels/canvas/models.py

```diff
@@ -91,6 +91,14 @@ class CanvasEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
         help_text="The base URL used for API requests to Canvas, i.e. https://instructure.com."
     )
 
+    refresh_token = models.CharField(
```

Comment:
About this field, we won't always have refresh token available when we create this model, it will be populated later right? so do we need to allow NULL in db?

Reply:
nvm, I think empty string is better for db, but I would also get Talia's opinion on this one